### PR TITLE
Add Canon Imageformula driver 1.6.18

### PR DESCRIPTION
**Pull request by captnswing**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask-drivers/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask-drivers/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/canon-imageformula-driver.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+cask 'canon-imageformula-driver' do
+  version '1.6.18.1009'
+  sha256 'c4bd8d730bc0066f47e9a5651a0051c5d8402e438f1899d2c7152376ad61ed05'
+
+  # gdlp01.c-wss.com/gds/1/0100009941/02 was verified as official when first introduced to the cask
+  url "http://gdlp01.c-wss.com/gds/1/0100009941/02/P-215IIDriverV.#{version}forMac.dmg"
+  name 'Canon ImageFormula Twain Driver'
+  homepage 'https://www.canon.se/support/consumer_products/products/scanners/others/imageformula_p-215ii.html?type=drivers'
+
+  depends_on macos: [
```
Comment (vitorgalvao) [inline Casks/canon-imageformula-driver.rb]: Why not `>= ':mojave'`? Is it specifically stated it won’t work in Catalina?

Comment (captnswing) [reply Casks/canon-imageformula-driver.rb]: I'm new to writing Casks :)

Sierra, High Sierra and Mojave are explicitly listed as supported on the homepage. 
So maybe `>= 'sierra'`?

As for Catalina support, no explicit statement on the homepage, and I have no way to test. But in my experience the vendor (Canon) has a pretty bad track record in supporting new MacOS versions. Usually they take a while and then release a new version

Comment (vitorgalvao) [reply Casks/canon-imageformula-driver.rb]: > So maybe `>= ':sierra'`?

Yes, that’s what I meant.

> in my experience the vendor (Canon) has a pretty bad track record in supporting new MacOS versions. Usually they take a while and then release a new version

Then it’s fine as is. Thank you.

**Comment by vitorgalvao** [comment]
@core-code Are you able to find an appcast for this one?

I was able to find one for https://github.com/Homebrew/homebrew-cask-drivers/pull/1176 (similar cask) by inspecting the network traffic (via Chrome’s Developer Tools), searching for the version string, opening that (long) URL and trimming it down to essentials.

No such luck with this case.

I don’t mind this not having an appcast if you also don’t, but since we’re trying to add appcasts from the start, it’s a good idea to discuss the trickier cases.

**Comment by core-code** [comment]
i think we can use 
`https://my.canon/en/support/0100992103/2`
both as a homepage (its english instead of swedish ;) and as an appcast

**Comment by captnswing** [comment]
...this could be merged? 😀

**Pull request merged**